README.md

```diff
@@ -611,20 +661,30 @@ class Sabberworm\CSS\CSSList\Document#4 (2) {
 
 ## Contributors/Thanks to
```

Comment:
I don’t like breaking up list items over multiple lines. Makes it hard to move them up or down in the editor (which is something I tend to do a lot more often with list items than with other paragraphs).

Reply:
Come to think of it, I also prefer to rely on the editor’s soft-wrap behaviour for paragraphs. But my opinions on that are not as strong as on list items.

Reply:
Rebased, changed and repushed.